libs/image/include/image/ColorTransform.h

```diff
@@ -198,6 +199,65 @@ std::unique_ptr<uint8_t[]> fromLinearToRGBM(const Image& image) {
     return dst;
 }
 
+template <typename T>
+std::unique_ptr<uint8_t[]> fromLinearTosRGB(const LinearImage& image) {
+    using math::float3;
+    size_t w = image.getWidth();
+    size_t h = image.getHeight();
+    size_t channels = image.getChannels();
```

Comment:
There's a mix of hard-coded `3` and `channels`.

Reply:
Done

Reply:
This was copied from old code but I added comments and assertions to clarify the input / output expectations for these functions.